tests/vad/test_webrtc.py

```diff
@@ -0,0 +1,36 @@
+"""
+This module contains tests for Voice Activity Detection
+"""
+from unittest.mock import patch
+
+import numpy as np
+
+from spokestack.context import SpeechContext
+from spokestack.vad.webrtc import VoiceActivityDetector
+
+
+@patch("spokestack.vad.webrtc.webrtcvad")
+def test_vad(mock_class):
+    context = SpeechContext()
+    detector = VoiceActivityDetector(
+        sample_rate=16000,
+        frame_width=10,
+        vad_rise_delay=0,
+        vad_fall_delay=20,
+        mode="quality",
+    )
+
+    # test detection
```

Comment:
this only tests that the vad is eventually triggered because the mock is always returning true, you'll also want to test the following states: 
. verify that the vad remains untriggered if vad_rise_delay hasn't been reached and the mock is returning true
. verify that the vad remains triggered if vad_fall_delay hasn't been reached and the mock is returning false
. verify that the vad becomes untriggered if vad_fall_delay has been reached and the mock is returning false